study-builder/studies/pancan/substitutions.conf

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Can we rename this file to `subs.conf`? (I really don't want to type out the whole thing...)

Reply:
I initially had it subs and just before push changed it